pointocracy.client/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "pointocracy.client",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
```

Comment:
Do we want to change these to be locked on a specific version?

Reply:
Yes